common/app/experiments/Experiments.scala

```diff
@@ -73,12 +72,3 @@ object EuropeNetworkFront
       sellByDate = LocalDate.of(2023, 3, 1),
       participationGroup = Perc0D,
     )
-
-object BillboardsInMerchSlots
```

Comment:
My original plan was to change the slots server-side but decide it was just simpler to do all this on the client. Hence we get rid of this test and introduce a traditional client-side one.